.github/guides/STYLES.md

```diff
@@ -382,6 +382,6 @@ set_invincible(FALSE) // Fine! Boolean parameters don't always need to be named.
 The following coding styles are not only not enforced at all, but are generally frowned upon to change for little to no reason:
 
 * English/British spelling on var/proc names
-	* Color/Colour - both are fine, but keep in mind that BYOND uses `color` as a base variable
```

Comment:
one PR with the same aim as already been merge.
the idea is to have only have one spelling to avoid confusion?
and be able to navigate the code either...
i am only changing it to American since the code in UK is a minority...

Reply:
I didn't knew the entire continent of america spoke english! That's good to know

Reply:
> I didn't knew the entire continent of america spoke english! That's good to know

I'm Canadian, we spell it Colour, it was changed by the Americans because of printing press limitations in the early days of the USA